agent/containers/images/visualizers/DockerfilePostProm

```diff
@@ -0,0 +1,18 @@
+# To be ran from parent 'visualize' directory: podman build -t <name> -f DockerfilePostProm .
```

Comment:
"To be run" ... although in this context, something like "Use this with the following command" might be more clear. Also, I assume you mean "'visualizers' directory" ... and "parent" seems superfluous and possibly confusing since the file is _in_ that directory.